axon-server/administration/monitoring/heartbeat-monitoring.md

```diff
@@ -1,30 +1,40 @@
 # Heartbeat Monitoring
 
-You can enable the heartbeat monitoring in Axon Server in order to activate an high level monitoring of client connection availability.‌
+You can enable the heartbeat monitoring in Axon Server in order to activate an high level monitoring of client connection availability.
 
-As gRPC already provides an internal heartbeat, this feature is to cover the scenarios for which the gRPC implementation does not suffice.‌
+As gRPC already provides an internal heartbeat, this feature is to cover the scenarios for which the gRPC implementation does not suffice.
 
 This feature can be enabled by configuring the following property:
```

Comment:
Since it's on be default, should we  not invert this, by telling how to disable it?

Reply:
Agreed

Reply:
Good point, will adjust.